tests/test_web_urldispatcher.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import tempfile
+import aiohttp
+from aiohttp import web
+import os
+import shutil
+import asyncio
+
+# Timeout in seconds for an asynchronous test:
+ASYNC_TEST_TIMEOUT = 1
+
+
+class ExceptAsyncTestTimeout(Exception):
+    pass
+
+
+def run_timeout(cor, loop, timeout=ASYNC_TEST_TIMEOUT):
```

Comment:
Do you really need the function? What the purpose of timeout parameter -- looks like you don't use it.